core/controllers/feedback_test.py

```diff
@@ -554,3 +556,214 @@ def test_feedback_threads_with_suggestions(self):
+            'state_name': 'Welcome!',
+            'new_value': new_content}
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION, self.EXP_ID,
+            1, suggestion_models.STATUS_IN_REVIEW, self.owner_id_1,
+            None, change, 'score category', thread_id)
+
+        thread_url = '%s/%s' % (feconf.FEEDBACK_THREAD_URL_PREFIX, thread_id)
+        self.post_json(
+            thread_url, {
+                'text': None,
+                'updated_subject': None,
+                'updated_status': None
+            }, csrf_token=csrf_token, expected_status_int=400)
+
+        self.logout()
+
+    def test_get_feedback_threads_with_updated_suggestion_status_raises_400(
+            self):
+        self.login(self.OWNER_EMAIL_1)
+        response = self.get_html_response('/create/%s' % self.EXP_ID)
+        csrf_token = self.get_csrf_token_from_response(response)
+
+        feedback_services.create_thread(
+            feconf.ENTITY_TYPE_EXPLORATION, self.EXP_ID, self.owner_id_1,
+            'a subject', 'some text', has_suggestion=True)
+
+        response_dict = self.get_json(
+            '%s/%s' % (feconf.FEEDBACK_THREADLIST_URL_PREFIX, self.EXP_ID))
+        threadlist = response_dict['suggestion_thread_dicts']
+        thread_id = threadlist[0]['thread_id']
+
+        new_content = state_domain.SubtitledHtml(
+            'content', 'new content html').to_dict()
+        change = {
+            'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+            'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+            'state_name': 'Welcome!',
+            'new_value': new_content}
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION, self.EXP_ID,
+            1, suggestion_models.STATUS_IN_REVIEW, self.owner_id_1,
+            None, change, 'score category', thread_id)
+
+        thread_url = '%s/%s' % (feconf.FEEDBACK_THREAD_URL_PREFIX, thread_id)
+        self.post_json(
+            thread_url, {
+                'text': 'Message 1',
+                'updated_subject': None,
+                'updated_status': 'open'
+            }, csrf_token=csrf_token, expected_status_int=400)
+
+        self.logout()
+
+
+class ThreadListHandlerForTopicsHandlerTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ThreadListHandlerForTopicsHandlerTests, self).setUp()
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.set_admins([self.OWNER_USERNAME])
+
+        self.topic_id = topic_services.get_new_topic_id()
+        self.save_new_topic(
+            self.topic_id, self.owner_id, 'Name', 'Description',
+            [], [], [], [], 1)
```

Comment:
Please use kwargs to help readers understand what each list represents.

Reply:
Again I don't think its better to use arg name for non keyword arguments. PTAL at my comments in other PRs. Thanks!

Reply:
FYI, we have a lint rule that argnames should only be used precisely when there is a default value specified, and not used otherwise.

Reply:
@seanlip you mean https://github.com/oppia/oppia/issues/6067 can be closed now?

Reply:
Sorry no, I meant that we *should* have a style check that implements this rule (which is why it's worth finishing up #6067).

Reply:
I added my thoughts to that issue, TL;DR: I strongly disagree about disallowing kwarg-names. Let's discuss more there?

Reply:
This issue is resolved now. See #6067 for more info